app/models/registry.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env ruby
```

Comment:
I think as that probably will be a gem at some point - it is better to settle it in the vendor